vault/resource_gcp_auth_backend.go

```diff
@@ -189,6 +239,16 @@ func gcpAuthBackendRead(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	if endpointsRaw, ok := resp.Data["custom_endpoint"]; ok {
+		endpoints, ok := endpointsRaw.(map[string]interface{})
+		if !ok {
+			return fmt.Errorf("custom_endpoint has unexpected type: %q", path)
```

Comment:
Suggest including the type in the error:
```suggestion
			return fmt.Errorf("custom_endpoint has unexpected type %T, path=%q", endpointsRaw, path)
```